protocol/scripts/deploys/mainnet/v2/post-deploy/temple-v2-executor-transactions-batch.json

```diff
@@ -0,0 +1,250 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1690182680732,
+  "meta": {
+    "name": "Temple Executor V2 & TLC",
+    "description": "Executor initial transactions",
+    "txBuilderVersion": "1.16.1",
+    "createdFromSafeADDRESS": "0x4D6175d58C5AceEf30F546C0d5A557efFa53A950",
+    "createdFromOwnerADDRESS": "",
+    "checksum": "0x34fd3c2dab040576a17db549ea9b158fd9836d0ecec4245c24082e50ef27e70e"
+  },
+  "transactions": [
+    {
+      "to": "0x470EBf5f030Ed85Fc1ed4C2d36B9DD02e77CF1b7",
```

Comment:
I don't think we need to addMinter to temple for mainnet, can you recall if/why we need it?

Reply:
am I interpreting some incorrectly 😅, I believe `TempleTokenBaseStrategy.sol` does run `templeToken.mint` and `templeToken.burn` and that's why we need to add it here?

Reply:
It does - ignore